src/shared/dm-util.h

```diff
@@ -1,4 +1,15 @@
 /* SPDX-License-Identifier: LGPL-2.1+ */
 #pragma once
 
+#include <stdlib.h>
+#include "macro.h"
+
 int dm_deferred_remove(const char *name);
+int dm_deferred_remove_cancel(const char *name);
+static inline void dm_deferred_remove_clean(char *name) {
+    if (!name)
+        return;
+    (void)dm_deferred_remove(name);
```

Comment:
doesn't matter, but we usually place a space between the `(void)` and the function name

Reply:
fixed